photo/tests/factories.py

```diff
@@ -17,6 +17,7 @@
 class UserFactory(factory.django.DjangoModelFactory):
     class Meta:
         model = User
+        skip_postgeneration_save = True
```

Comment:
@JdFSilva questioned why this was added

Reply:
This was to resolve a deprecation warning that was being raised by all factories using post_generation.

Reply:
See the deprecation section of https://github.com/FactoryBoy/factory_boy/blob/master/docs/changelog.rst#330-unreleased